### ISSUE TITLE
text not working

### ISSUE THREAD

**Issue by supervoidcoder** [PenguinAI-Ext/extension#6]
I can generate images using this, but using any type of text generation models seems to wait for like 1 minute and then it just gives a Networkerror. Even the API by abby says it's "online" but it's clearly mostly dead. I thought it was a CORS issue at first since I tried using it for my scratch ai assistant at greencuby.com/scratchai but it's still not working in penguinmod itself, even with a simple prompt (since i thought the relatively long prompt could have also been the issue).

**Comment by LiamUzap** [comment]
Also problems here, I cant even launch the extension, it throws this error (USING THE .js FILE)
![Screenshot 2024-11-24 150427](https://github.com/user-attachments/assets/1d3fe9d8-ff9d-4380-84e7-21bd354b1d60)

(I have not added the thumbnail or anything other then the code just to test if it was legit)
(I have tried not using sandbox)
(I am using the DuckDuckGo Browser)
So, can anybody help me with this error, Thanks

**Comment by LiamUzap** [comment]
I have tried many browsers none work

**Comment by supervoidcoder** [comment]
I think it IS a cors issue, actually. In the console of PenguinMod itself, it says "Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://penguinai.milosantos.com/v1/chat/completions. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing). Status code: 524.

Error sending prompt to GPT NetworkError when attempting to fetch resource."

**Comment by cicerorph** [comment]
Hey @supervoidcoder, could you try using https://api.penguinai.tech/v1 as the base reverse proxy url?

**Comment by supervoidcoder** [comment]
> Hey @supervoidcoder, could you try using https://api.penguinai.tech/v1 as the base reverse proxy url?

Sure, I'll try!

**Comment by supervoidcoder** [comment]
> Hey @supervoidcoder, could you try using https://api.penguinai.tech/v1 as the base reverse proxy url?

Nope, none of the following work, they return false with the boolean, and say "NetworkError when attempting to fetch resource.":
https://proxy.mubilop.tech/v1
https://api.penguinai.tech/v1
https://penguinai.milosantos.com/v1

 Error 1033 Argo tunnel error

**Comment by 1TSnakers** [comment]
I use https://penguinai.derpygamer2142.com/v1; it is the same as https://proxy.mubilop.tech/v1, but it works (I have it running next to me as I am writing this).

Most of the popular models aren't working (usually the ones on the bottom), but the rest do!

**Comment by cicerorph** [comment]
> > Hey @supervoidcoder, could you try using https://api.penguinai.tech/v1 as the base reverse proxy url?
> 
> Nope, none of the following work, they return false with the boolean, and say "NetworkError when attempting to fetch resource.": https://proxy.mubilop.tech/v1 https://api.penguinai.tech/v1 https://penguinai.milosantos.com/v1
> 
> Error 1033 Argo tunnel error

Try again please.

**Comment by cicerorph** [comment]
Oof missclick

**Comment by TNT-TheGamerFnfBeing** [comment]
> I use https://penguinai.derpygamer2142.com/v1; it is the same as https://proxy.mubilop.tech/v1, but it works (I have it running next to me as I am writing this).
> 
> Most of the popular models aren't working (usually the ones on the bottom), but the rest do!

It says the proxy is working now, but none of the chatbots work.
I'm currently going through them as I type this out.

**Comment by supervoidcoder** [comment]
yeah not working now, nothing's working now
